vsts/sdl.yaml

```diff
@@ -69,7 +69,7 @@ phases:
       FileDirPath: '$(Build.StagingDirectory)'
       TreatSignatureUpdateFailureAs: 'Warning'
       SignatureFreshness: 'UpToDate'
-      TreatStaleSignatureAs: 'Error'
```

Comment:
In the event that the antimalware executable fails to update its signature, we shouldn't fail the SDL run

Reply:
What does it mean for the antimalware executable to fail to update its signature? 
Under what circumstances does that happen?

Reply:
> What does it mean for the antimalware executable to fail to update its signature? 
It means that whatever server this task requests the latest signature from fails to deliver it for whatever reason. Possibly due to server errors

> Under what circumstances does that happen?
We've only started seeing this recently, so I'd chalk it up as a server error for whatever service manages these signature update requests

Reply:
The main reason I ask is to ensure that ignoring them is safe

Reply:
My understanding is that this is safe since the signature will get updated eventually even if it fails once or twice in a row. Would you prefer I leave the antimalware task alone, though?

Reply:
Honestly, unless we fully know the impact of this signature getting updated, I would rather keep it as an error until we know more.

Reply:
Okay, I'll revert this particular change